### DISCUSSION TITLE
Programmatically load 3D Tiles

### DISCUSSION THREAD

**Discussion by BL-JrmF** [iTowns/itowns#2319] [Q&A]
Hi,

I am currently working on a "3D feature viewer" using ITowns `PlanarView` containing multiple `C3DTilesLayer`, each containing set of tiles.
I searched for a way to 'programmatically' tell a layer to load it's tiles and tile content, in order to interact with the objets within the tiles once it's loaded.

I have found in the documentation that `C3DTilesLayer` trigger an event `on-tile-content-loaded` when the camera do focus the layer coordinates and the tiles appear at the screen, but this is not what I am looking for.

Is there a way to load a specific layer so the set of tile and it's content are ready to be used ?

**Comment by jailln** [comment]
Hi,
It is currently not possible. Tiles are only loaded when they should be displayed.

The only workaround that I can think of (although it won't cover all cases) would be to set the `sseThreshold` parameter of the layer to a small value which will lead to a more agressive subdivision and therefore to load more child tiles. However it won't load tiles that are out of the camera frustum. 

What is your use case?
